### ISSUE TITLE
Fix Custom Tunic Texture Mod Incompatibility Issues

### ISSUE THREAD

**Issue by DiabloThePoet** [gamestabled/OoT3D_Randomizer#532]
Myself and others that I've talked to and seen have been having big issues with imported custom textures from mods that have to do with Link's tunic just not working when the randomizer is activated. They work fine on regular files with the randomizer disabled, which leads me to believe it has something to do with the randomizer. 

Custom imported mod textures on shields, swords, hair, Link's face, Link's sleeves, and Link's boots all work fine. It's just his tunic and hat that do not seem to work at all. Will just show up as the default colors, or whatever custom color was entered into the randomizer, or sometimes just shows up as a blank grey texture. This is all even with custom tunic colors turned off in the randomizer menu.

This may have something to do with the randomizer's built-in function to change Link's tunic color overriding any outside texture changes from mods. Not sure, though. Hopefully, this can be fixed because I'd love to be able to use custom tunic textures from imported mods with the randomizer, and not just be limited to the options within the cosmetic menu!

**Comment by LinkexaYT** [comment]
The custom tunic textures work fine for me, it might just be you, mate.

**Comment by DiabloThePoet** [comment]
I'm referring to imported tunic texture packs from mods not working, not the custom colors you can set the tunics to inside of the cosmetic menu on the randomizer. The cosmetic menu works fine and I can set the tunics to any color I want using that. That's seemingly all I can do, though.

**Comment by Kewlan** [comment]
Fixed by #605. Disable custom tunic colors and you can use these mods. 👍

**Comment by DiabloThePoet** [comment]
Kewlan, It very briefly worked for me, but is now no longer working, even when I have custom tunic colors off. I updated to the latest version of Nightly, and it was working. I then updated luma, generated a new seed, received an error message, and now it is just no longer working. I'm back to getting the grey tunic textures. I've tried reverting my luma back to what I had before, preventing the randomizer to previous versions post the "fixed version", deleting and redownloading the texture pack, but nothing is working. I'm at a total loss.
